src/NServiceBus.Core/Sagas/IMessagePropertyMapper.cs

```diff
@@ -0,0 +1,18 @@
+namespace NServiceBus
+{
+    using System;
+    using System.Linq.Expressions;
+
+    /// <summary>
+    /// A helper class that proved syntactical sugar as part of <see cref="SimpleSaga{TSagaData}.ConfigureHowToFindSaga(IConfigureHowToFindSagaWithMessage)" />.
+    /// </summary>
+    public interface IMessagePropertyMapper
```

Comment:
ie we can have this

    protected override void ConfigureMapping(IMessagePropertyMapper mapper)

instead of

    protected override void ConfigureMapping(MessagePropertyMapper<SagaData> mapper)